backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -487,12 +488,19 @@ open class SubmissionDatabaseService(
 
         val groupCondition = getGroupCondition(groupIdsFilter, authenticatedUser)
 
+        val submitterCondition = if (submitterNamesFilter !== null) {
+            SequenceEntriesView.submitterIsOneOf(submitterNamesFilter)
+        } else {
+            Op.TRUE
+        }
+
         val organismCondition = SequenceEntriesView.organismIs(organism)
 
         val accessionVersionsToUpdate = SequenceEntriesView
             .selectAll()
             .where {
-                statusCondition and accessionCondition and scopeCondition and groupCondition and organismCondition
+                statusCondition and accessionCondition and scopeCondition and groupCondition and organismCondition and
```

Comment:
As there might be more conditions in the future, would be kind of nice to make them a list like this 

```
statusCondition and 
accessionCondition and 
scopeCondition and 
groupCondition and 
organismCondition and
```